### PR TITLE
feat: Remove option to select protocol in create group - WPB-10442

### PR DESCRIPTION
**Pull request by KaterinaWire**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10442" title="WPB-10442" target="_blank">BugWPB-10442</a>  Remove option to select protocol in create group
  </td></table>
  <br />
 

### Issue

By default, we should not see the selection of protocols when creating a conversation. However, this should be possible for testing.

### Testing

By default, the option to select a protocol when creating a group should be hidden.

### Needs to release with
- [ https://github.com/wireapp/wire-ios-build-assets/pull/130 ]

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [ ] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [ ] Make sure you use the API for UI elements that support large fonts.
- [ ] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [ ] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift [hunk 1]
```diff
@@ -113,7 +113,6 @@ final class ConversationCreationController: UIViewController {
 
     private lazy var encryptionProtocolSection = {
         let section = ConversationEncryptionProtocolSectionController(values: values)
-        section.isHidden = true
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift]: This was not fixed during the conversation creation view refactoring.

**Review by netbe — COMMENTED** [review]
I don't see removing MLS_ENABLED following the PR in wire-ios-build-assets + updating the submodule ref.
@KaterinaWire do you plan another PR for this?

**Comment by KaterinaWire** [comment]
> I don't see removing MLS_ENABLED following the PR in wire-ios-build-assets + updating the submodule ref. @KaterinaWire do you plan another PR for this?

@netbe I'm removing MLS_ENABLED in https://github.com/wireapp/wire-ios/pull/2129

The submodule ref for `wire-ios-build-assets `is updated in https://github.com/wireapp/wire-ios/pull/2129

File: wire-ios-utilities/Source/DeveloperFlag.swift [hunk 2]
```diff
@@ -87,7 +87,7 @@ public enum DeveloperFlag: String, CaseIterable {
         case .enableMLSSupport:
             return "MLSEnabled"
         case .showCreateMLSGroupToggle:
-            return "CreateMLSGroupEnabled"
+            return nil
```
Comment (samwyndham) [inline wire-ios-utilities/Source/DeveloperFlag.swift]: OK so returning nil here means it is still available for developer builds but not for regular builds?

Comment (KaterinaWire) [reply wire-ios-utilities/Source/DeveloperFlag.swift]: @samwyndham  Yes

**Review by samwyndham — APPROVED** [review]

**Review by jullianm — APPROVED** [review]

File: wire-ios-utilities/Resources/WireUtilities-Info.plist [hunk 3]
```diff
@@ -24,8 +24,6 @@
 	<string>Copyright © 2015 Wire. All rights reserved.</string>
```
Comment (netbe) [inline wire-ios-utilities/Resources/WireUtilities-Info.plist:24]: missing ref of submodule [d48bf72](https://github.com/wireapp/wire-ios-build-assets/commit/d48bf725061aabfd2f8d180240d10fdefd81d7f7)

**Review by netbe — APPROVED** [review]
approving but left a comment not to forget to update the submodule ref

File: wire-ios-utilities/Resources/WireUtilities-Info.plist (hunk 3 above)
Comment (KaterinaWire) [reply wire-ios-utilities/Resources/WireUtilities-Info.plist:24]: It was updating in the PR I was expecting to merge first, but I updated it here too

**Pull request merged**